test/test_response.py

```diff
@@ -65,10 +65,21 @@ def test_getheader(self) -> None:
 
 class TestResponse:
     def test_cache_content(self) -> None:
-        r = HTTPResponse(b"foo")
+        r = HTTPResponse(b"foo", decode_content=False)
```

Comment:
Is this a behavior change?

Reply:
No, this is irrelevant because this tests the cache set at `__init__` time. Removed: [`86b6b53` (#2712)](https://github.com/urllib3/urllib3/pull/2712/commits/86b6b5378e132f86aed9cd242ea86e10acd04879)